src/ontology/mondo-edit.obo

```diff
@@ -223276,12 +223274,8 @@ xref: OMIM:603543 {source="MONDO:equivalentTo", source="Orphanet:69085", source=
 xref: Orphanet:69085 {source="OMIM:603543", source="MONDO:equivalentTo"}
 xref: SCTID:721972001 {source="MONDO:equivalentTo"}
 xref: UMLS:C1863753 {source="OMIM:603543", source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset", source="Orphanet:69085", source="Orphanet:69085/e"}
-is_a: MONDO:0015853 {source="Orphanet:69085"} ! deficient breast volume or number
-is_a: MONDO:0018234 ! dysostosis
-is_a: MONDO:0018454 {source="MONDO:0018455-obsoleted"} ! dysostosis of genetic origin
-is_a: MONDO:0019054 ! congenital limb malformation
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditary disease
```

Comment:
I don't think 'limb-mammary syndrome' should be a breast disorder, as that is a feature:

see OMIM description

Limb-mammary syndrome (LMS) is an autosomal dominant disorder characterized by variable expressivity of severe hand and/or foot anomalies (deficiencies, duplications, and fusion/separation defects) and hypoplasia/aplasia of the mammary gland and nipple. Less frequent findings include lacrimal duct atresia, nail dysplasia hypohidrosis, hypodontia, and cleft palate with or without bifid uvula